eva-lib/src/test/java/uk/ac/ebi/eva/lib/repository/VariantEntityRepositoryTest.java

```diff
@@ -74,13 +75,7 @@ public void checkFieldPresence() throws IOException {
         List<String> exclude = new ArrayList<>();
 
         List<VariantEntity> variantEntityList = variantEntityRepository
-                .findByRegionsAndComplexFilters(regions, null, null,
-                                                VariantEntityRepository.RelationalOperator.NONE,
-                                                null,
-                                                VariantEntityRepository.RelationalOperator.NONE,
-                                                null,
-                                                VariantEntityRepository.RelationalOperator.NONE,
-                                                null, exclude,
+                .findByRegionsAndComplexFilters(regions, new ArrayList<>(), exclude,
```

Comment:
Make the second argument a variable for clarity (same as `regions` and `exclude`).